src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1431,6 +1431,7 @@ public static string WrongColumnDimensions(string columnName, int xDim, int yDim
       public static readonly string TransportMoleculeNamesNotSet = "Transport molecule names object passed is not set";
       public static readonly string TransportMoleculeNamesBothListsNonEmpty = "Molecule names to transport and molecule names not to transport are both nonempty";
       public static readonly string InvalidFile = "Invalid File";
+      public static readonly string InvalidAuxiliaryType = "The error type is invalid. Valid types are ArithmeticStdDev and GeometricStdDev";
```

Comment:
Can you put ArithmeticStdDev and GeometricStdDev in quote e.g. 'GeometricStdDev'

Reply:
Sure ;)